app/views/html_attachments/_print_meta_data.html.erb

```diff
@@ -0,0 +1,40 @@
+<br/>
+<br/>
+<br/>
+<p>
+<%= image_tag "open-government-licence.png", id: "meta-data-licence" %>
+</p>
+<br/>
+<br/>
+<br/>
+<p>
+  © Crown copyright <%= Date.today.year %>
```

Comment:
I've checked with Theo, who wrote the requirements. He said it makes sense to have it as the year when the html attachment was created.